source/common/stats/symbol_table.cc

```diff
@@ -14,13 +14,6 @@
 namespace Envoy {
 namespace Stats {
 
-// Masks used for variable-length encoding of arbitrary-sized integers into a
```

Comment:
The changes in this file are to move the code for SymbolTable::Encoding into its own standalone class in source/common/common/utility.h so the encoding mechanism can be shared by TinyString. There is no functional change to the symbol table implementation; just renaming/moving the helper class.